lib/spoom/context.rb

```diff
@@ -0,0 +1,215 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "fileutils"
+require "open3"
+
+module Spoom
+  # An abstraction to a Ruby project context
```

Comment:
```suggestion
  # An abstraction to a Ruby project context.
```

Reply:
I think by now most of Spoom is documented without a `.`. I'll do a pass on the documentation at some point.